docs/antora/modules/ROOT/pages/Intro_to_Mill.adoc

```diff
@@ -912,7 +912,15 @@ object foo extends ScalaModule with ScalaMetalsSupport {
   def scalaVersion = "{example-scala-version}"
 }
 ----
---
+=== Using Bloop (standalone BSP server)
```

Comment:
You need an extra empty line here

```suggestion

=== Using Bloop (standalone BSP server)
```

Reply:
Thankyou

Reply:
Oh, it's not the missing extra line but the line with the two dashes (`--`) which you removed before, that's missing here.

Reply:
```suggestion
--

=== Using Bloop (standalone BSP server)
```